submit resnet101 inference on alibaba npu

raoxing opened the pull request:
update alipay resnet101 on npu

raoxing committed:
Create Alipay_resnet101_npu_tensorflow.json

update alipay resnet101 on npu

codyaustun commented on ImageNet/inference/Alipay_resnet101_npu_tensorflow.json:
```diff
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "Ai CognitiveComputing team in Alipay Group",
```
This might be a little cleaner:

```suggestion
    "author": "AI Cognitive Computing team in Alipay Group",
```

codyaustun commented on ImageNet/inference/Alipay_resnet101_npu_tensorflow.json:
```diff
@@ -0,0 +1,22 @@
+    "authorEmail": "xing.rao@alibaba-inc.com, weihong.wh@alibaba-inc.com",
+    "framework": "tensorflow+NpuInference",
+    "codeURL": "https://github.com/alipay-cognition/DAWNBench_Inference",
+    "commitHash": "",
```
Please don't leave the `commitHash` field blank. Assuming you want the most recent commit on master, it should be:

```suggestion
    "commitHash": "ea76aaa2699e2dd1fdf46ae144c332f83f4b3d9c",
```

codyaustun commented on ImageNet/inference/Alipay_resnet101_npu_tensorflow.json:
```diff
@@ -0,0 +1,22 @@
+    "codeURL": "https://github.com/alipay-cognition/DAWNBench_Inference",
+    "commitHash": "",
+    "model": "ResNet101",
+    "hardware": "Alibaba Cloud Npu, from https://promotion.aliyun.com/ntms/act/alinpu.html?spm=5176.12901015.0.i12901015.4c27525cDE1r6q",
```
This is meant to be a short name for the hardware. Please remove the URL. If you want to include it, add it under the `misc` field.

codyaustun reviewed (changes requested):
Thanks for the submission! I just have a few small requests for the JSON file.

raoxing committed:
Update Alipay_resnet101_npu_tensorflow.json

Fix some problems in https://github.com/stanford-futuredata/dawn-bench-entries/pull/126/commits/6512efaafed245efadca3aeb0a351953b28422f5

codyaustun commented:
Looks good to me. Thanks!
